Makefile

```diff
@@ -47,27 +58,34 @@ else
 	LIBDEFLATE=libdeflate.a
 endif
 
-%.o: %.cpp
+SRC_7ZIP = $(OBJDIR)/7zip/7zip.a
+SRC_LIBDEFLATE = $(SRCDIR)/libdeflate/$(LIBDEFLATE)
```

Comment:
Sorry, almost missed this - does this need to be OBJDIR?  Or maybe libdeflate still isn't allowing out of tree builds, itself?

-[Unknown]

Reply:
Yea, libdeflate does not support it yet and its object files unfortunately end up in the wrong place. I was planning to try to look at that more closely later, but your Makefile is much easier to work with. For the sake of distros it just might be easier to use a system version of libdeflate.